iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_sender.inl

```diff
@@ -147,30 +147,51 @@ template <typename ChunkSenderDataType>
 inline void ChunkSender<ChunkSenderDataType>::send(mepoo::ChunkHeader* const chunkHeader) noexcept
 {
     mepoo::SharedChunk chunk(nullptr);
-    // BEGIN of critical section, chunk will be lost if process gets hard terminated in between
+    // BEGIN of critical section, chunk will be lost if the process terminates in this section
     if (getChunkReadyForSend(chunkHeader, chunk))
     {
         this->deliverToAllStoredQueues(chunk);
 
         getMembers()->m_lastChunkUnmanaged.releaseToSharedChunk();
         getMembers()->m_lastChunkUnmanaged = chunk;
     }
-    // END of critical section, chunk will be lost if process gets hard terminated in between
+    // END of critical section
+}
+
+template <typename ChunkSenderDataType>
+inline bool ChunkSender<ChunkSenderDataType>::sendToQueue(mepoo::ChunkHeader* const chunkHeader,
+                                                          const cxx::UniqueId uniqueQueueId,
+                                                          const uint32_t lastKnownQueueIndex) noexcept
+{
+    mepoo::SharedChunk chunk(nullptr);
+    // BEGIN of critical section, chunk will be lost if the process terminates in this section
+    if (getChunkReadyForSend(chunkHeader, chunk))
```

Comment:
What happens when `chunkHeader == nullptr`?

Reply:
Then `getChunkReadyForSend` will return false and the error handler will be called